app/services/web-development.html

```diff
@@ -105,6 +105,33 @@ <h2 class="title my-4">Websites</h2>
           Media setup, Business Emails setup and Search Engine Optimization we create an all-round solution to set the
           ball rolling for your business.
         </p>
+
+        <div class="d-flex justify-content-between align-items-center flex-wrap">
+          <picture>
+            <img src="../images/services/bootstrap-logo.svg" alt="bootstrap logo" title="Bootstrap" class="img-fluid">
+          </picture>
+          <picture>
+            <img src="../images/services/sass.svg" alt="sass logo" title="Sass" class="img-fluid">
+          </picture>
+          <picture>
+            <img src="../images/services/wordpress-logo.svg" alt="wordpress logo" title="Wordpress" class="img-fluid">
+          </picture>
+          <picture>
+            <img src="../images/services/gulp-logo.svg" alt="gulp logo" title="Gulp" class="img-fluid">
+          </picture>
+        </div>
+        <div class="d-flex justify-content-between align-items-center flex-wrap mt-4">
+          <picture>
+            <img src="../images/services/bower-logo.svg" alt="bower logo" title="Bower" class="img-fluid">
+          </picture>
+          <picture>
+            <img src="../images/services/gatsbyjs-logo.svg" alt="gatsby logo" title="Gatsby" class="img-fluid">
```

Comment:
Correct capitalization in `alt`

Reply:
Fixed this typo in alt & title attribute